### ISSUE TITLE
Not compatible with CtrlP

### ISSUE THREAD

**Issue by genspire** [wellsjo/wellsokai.vim#2]
Using the [ctrlp](https://github.com/ctrlpvim/ctrlp.vim) vim plugin I am not able to see the file selection arrow.  There are other color schemes that do support it so the arrow show up.  

Would it be possible to update this color scheme to be compatible? 
Steps:
1. open up a vim (with ctrlp installed)
2. hit c-p
3. go up and down either using arrows or c-j and c-k

**Comment by wellsjo** [comment]
I agree, ctrlp should be compatible!  I didn't notice because I don't use that plugin.  I'll hopefully get to it soon. :smile:

**Comment by hrj** [comment]
I can see the arrow in the ctrlp's file selection window.

However, I can't see the current line highlight which is enabled using `:set cursorline`. Perhaps these two are related issues?
